packages/cli/src/services/user.service.ts

```diff
@@ -250,10 +246,7 @@ export class UserService {
 				async (transactionManager) =>
 					await Promise.all(
 						toCreateUsers.map(async ({ email, role }) => {
-							const newUser = Object.assign(new User(), {
-								email,
-								globalRole: role === 'member' ? memberRole : adminRole,
-							});
+							const newUser = transactionManager.create(User, { email, role });
```

Comment:
True. but I don't want to duplicate the input validation into the service layer as well. 
Any suggestion on what we could do? We do have string typing and unit tests to prevent this from happening. But maybe there is more that we could do?

Reply:
I think it's probably fine for now, I'm not too concerned about it, I just thought I'd raise it in case you had any ideas.